config/routes.rb

```diff
@@ -768,6 +768,8 @@
       resource :notifications, only: %i[show update], path: 'notification-settings'
     end
 
+    resources :organisation_settings, path: '/organisation-settings', only: :index
```

Comment:
Is the `path` required here? What does it produce if we don't specify?

Reply:
/provider/organisation_settings :)